desktop/package/macosx/copy_dbs.sh

```diff
@@ -2,12 +2,14 @@
 
 cd $(dirname $0)/../../../
 
+version="1.5.0"
+
 # Set BISQ_DIR as environment var to the path of your locally synced Bisq data directory e.g. BISQ_DIR=~/Library/Application\ Support/Bisq
 
 dbDir=$BISQ_DIR/btc_mainnet/db
 resDir=p2p/src/main/resources
 
-cp "$dbDir/TradeStatistics2Store" "$resDir/TradeStatistics2Store_BTC_MAINNET"
+cp "$dbDir/TradeStatistics3Store" "$resDir/TradeStatistics3Store_${version}_BTC_MAINNET"
```

Comment:
We do not update with each version (pre releases/hotfixes) the TradeStatistics3Store....

Reply:
I'm aware of it. I just wanted to add it to the script, so I don't have to copy it manually. I'll just not commit it if there is no intention to add a new trade stats store version to the release.

Reply:
I've added an additional comment to the script to make it more clear.